src/main/java/schemadefiner/retrieveschema/Mapper/SnowflakeMapper.java

```diff
@@ -0,0 +1,82 @@
+package schemadefiner.retrieveschema.Mapper;
+
+import schemacrawler.schema.Column;
+import schemacrawler.schema.Table;
+import schemadefiner.retrieveschema.GlueCrawler;
+import schemadefiner.retrieveschema.model.*;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class SnowflakeMapper {
+    public static List<TableDefinition> mapSchemaCrawlerTables(List<Table> tables, List<UserDefinedTable> userDefinedTables) {
+        //TODO: implement logic for userDefinedTables
+        return tables.stream().map(table -> mapSchemaCrawlerTable(table)).collect(Collectors.toList());
+    }
+
+    private static TableDefinition mapSchemaCrawlerTable(Table table) {
+        //TODO: sort the columns
+        List<Column> columns = table.getColumns();
+        List<Column> parsedPrimaryKeys = columns.stream().filter(c->c.isPartOfPrimaryKey()).collect(Collectors.toList());
```

Comment:
In the existing implementation, we sort tables by name:

https://github.com/phdata/streamliner/commit/d9442bd034fd18f677421d3f639bd073777b7d54#diff-2524c490aa18dfaaaabb0e7b3c59f674d27247579b47c7fc6cf8f1058d5f51a3R16

and columns by ordinal value:

https://github.com/phdata/streamliner/commit/d9442bd034fd18f677421d3f639bd073777b7d54#diff-2524c490aa18dfaaaabb0e7b3c59f674d27247579b47c7fc6cf8f1058d5f51a3R28

Should be able to do the same where by creating a custom Comparator using Comparator.comparing.